main.py

```diff
@@ -1,10 +1,52 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main():
-    pass
+    with open("players.json") as file_read:
+        players = json.load(file_read)
+
+    for player in players:
+        players_data = players[player]
+
+        #  guild
+        if players_data["guild"] is not None:
+            if not Guild.objects.filter(
+                    name=players_data["guild"]["name"]).exists():
+                Guild.objects.create(
+                    name=players_data["guild"]["name"],
+                    description=players_data["guild"]["description"])
+                players_guild = Guild.objects.get(
```

Comment:
This line should be out of the "if" statement so you can get the guild object when it already exists.

Reply:
Thanks! Fixed